packages/fxa-settings/src/components/Settings/FlowContainer/index.stories.tsx

```diff
@@ -27,3 +28,91 @@ export const Default = () => (
     {MOCK_CONTENT}
   </FlowContainer>
 );
+
+export const WizardExample = () => {
+  const [currentStep, setCurrentStep] = useState(1);
+  /*
+    In this wizard pattern, you'll want to have equality checks which control which step is being displayed (like,
+      { currentStep === 1 && (<YourViewHere/> )}
+    ).
+    Each step will be built with the <FlowContainer /> component as the outer wrapper.  Each step will at minimum expect the props `currentStep` and `setCurrentStep`, received from this component. The `currentStep` value and `setCurrentStep` method will be passed in to the `<FlowContainer/>` as props of the same name. Each step will want to have the <ProgressBar /> component as a child.
+    We have filler components hardcoded in here (the Wizard pattern seems too simple to justify making it into a reusable component), but we can replace these as we create the real views.
+    Each step should be a separate component.
+  */
+  return (
+    <>
+      {currentStep === 1 && (
+        <FlowContainer
```

Comment:
With the new UX flow, we'll want to pass in the title of the _previous_ view too to use as the back button title. Should we add that into the wizard demo?

Reply:
ohhhh interesting! That sounds like a modification that might need to happen to the entire FlowContainer component...should I work to include that modification in this PR?

Reply:
I think I've found a quick solution for this! Let me know what you think!

Reply:
Nice! Once we implement this in pages, we could pass in a string that's already localized with `ftlMsgResolver`, but I don't know if there's a huge advantage to that other than reducing the number of props that are passed to the FlowContainer.